bin/make_monthly_plot

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python
+
+import os
+from desisurvey import forecast
+from importlib import reload
+import matplotlib.pyplot as plt
+
+from desiutil.log import get_logger
+log = get_logger()
+
+from argparse import ArgumentParser
+ap = ArgumentParser(description=("Generates monthly status plot. For example:"
+                    "make_monthly_plot 20240725 20240826 $SCRATCH")
+                    )
+ap.add_argument("nightbegin",
+                help=("First night of interest (YYYYMMDD). Typically a month "
+                "before nightend.")
+                )
+ap.add_argument("nightend",
+                help=("Final night of interest (YYYYMMDD). Typically today.")
+                )
+ap.add_argument("desisurveyoutput",
+                help=("Directory to write plots and ephemeris files")
+                )
```

Comment:
Note that if you point this directory to somewhere that already has an ephemerides file, it will use that, rather than regenerating one.  Otherwise this is the longest step by far and everything else is fast.

Reply:
The code is pretty fast even without the cache! Regardless, I've added a note to the help message indicating that the ephemerides file is cashed to the `desisurveyoutput` directory for added speed.